mock-mhs-service/.gitignore

```diff
@@ -0,0 +1,37 @@
+HELP.md
```

Comment:
Do we need this .gitignore? All of these should be covered by the one in the main project dir